servers/cromwell/jobs/__main__.py

```diff
@@ -67,6 +68,10 @@
 try:
     with open(capabilities_path) as f:
         capabilities_config = json.load(f)
+    for settings in capabilities_config['displayFields']:
+        if settings['field'].startswith('labels.'):
+            if len(settings['field']) == 7 or len(settings['field']) > 262:
```

Comment:
nit: I would define static variables
```python
CROMWELL_LABEL_MIN_LENGTH = 1
CROMWELL_LABEL_MAX_LENGTH = 256
LABELS_PREFIX_LENGTH = len('.labels')
```
at the top of this file, and refactor this line to:
`if not CROMWELL_LABEL_MIN_LENGTH < len(settings['field']) - LABELS_PREFIX_LENGTH < CROMWELL_LABEL_MAX_LENGTH:`

But this is totally optional.

Reply:
apologies -- this PR is premature, as it relies on label editing being fully implemented, which it isn't